### DISCUSSION TITLE
Can't log in to my account

### DISCUSSION THREAD

**Discussion by jackbouncy7** [community/community#142222] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Hi everyone, my GitHub account was secured with an Authenticator app on my old phone. At the moment, I can't log in to my account, as it keeps asking for verification through the Authenticator app. I'm now using a new phone and having trouble logging back into my account. thank you!
my old profile @akbarahmadjonov

**Comment by ebndev** [comment, marked as answer]
Hi @jackbouncy7,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
